src/views/InternalsPage.tsx

```diff
@@ -70,6 +71,32 @@ class InternalsPage extends React.Component<Properties & WithStyles<typeof style
         this.state = {
             currentTime: props.initialTime
         };
+
+        this.startAsyncTimeChange(props.initialTime);
+    }
+
+    /**
+     * Set timeout to start minute update at correct offset from given moment
+     */
+    @bind
+    startAsyncTimeChange(date: moment.Moment): void {
```

Comment:
Je wilt de timer waarschijnlijk starten in `componentDidMount`, en de boel sowieso stopzetten in `componentWillUnmount`.  Met de huidige code blijft de timer doorgaan (met een referentie naar `this`) nadat het component weg is, waardoor dat geheugen nooit vrijgegeven kan worden.